iOS/PacketTunnelProvider/NetworkProtection/NetworkProtectionPacketTunnelProvider.swift

```diff
@@ -444,61 +445,40 @@ final class NetworkProtectionPacketTunnelProvider: PacketTunnelProvider {
 
         var tokenHandler: any SubscriptionTokenHandling
         var entitlementsCheck: (() async -> Result<Bool, Error>)
-        Self.isAuthV2Enabled = settings.isAuthV2Enabled
```

Comment:
From here to line 534 is just the usual subscription init swapped around V1<>V2 and the new `AuthMigrator `